advanced/wallets/react-wallet-v2/src/views/SessionSendTransactionModal.tsx

```diff
@@ -1,21 +1,19 @@
-import { Fragment, useState } from 'react'
-import { Divider, Modal, Text } from '@nextui-org/react'
+/* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Why?

Reply:
fixed it. The initial modal setup required rework of the modal parameters as it was causing conditional hooks usage